plugins/plugin-java/che-plugin-java-ext-lang-client/src/test/java/org/eclipse/che/ide/ext/java/client/organizeimports/OrganizeImportsPresenterTest.java

```diff
@@ -1,253 +0,0 @@
-/*
```

Comment:
Did we lose test coverage here?

Reply:
Unfortunately gwt tests are not run anymore in che. :(